public/app/features/admin/ActionButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { css, cx } from 'emotion';
+import { stylesFactory, useTheme } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    button: css`
+      text-decoration: underline;
+      color: ${theme.colors.blue95};
```

Comment:
use the external link color so it's consistent with the other link action on the page